wsgi.py

```diff
@@ -35,14 +45,30 @@ def application(environ, start_response):
 
     if URI_PATH.startswith(STATIC_URL):
         print('STATIC FILE DETECTED!')
+        print(URI_PATH)
+        file = str(URI_PATH).replace(STATIC_URL, "", 1)
+        requested_path = os.path.join(startdir, file)
+        requested_path = os.path.abspath(requested_path)
+        print(requested_path)
+        if not requested_path.startswith(startdir):
+            raise OSError("Hack")
+        headers[0] = ('Content-type', mimetypes.guess_type(URI_PATH, strict=True)[0]+'; charset=utf-8')
+        try:
+            f = open(requested_path, 'rb')
+        except FileExistsError:
+            status = '404 Not Found'
```

Comment:
If FileExistsError be raised then you change status to 404, after that you change status to 200 and do `body = f.read()` file. I think this is the wrong behavior.